README.md

```diff
@@ -6,20 +6,23 @@ It's purpose is to serve entities from a multitude of backends.
 
 ## Install the service
 
-First, download and install the Python package from GitHub:
+The service can be installed from SINTEF's GitLab:
+
+```shell
+export PIP_INDEX_URL=https://gitlab.sintef.no/api/v4/projects/17883/packages/pypi/simple
+pip install ds-entities-service
+```
+
+For development, we recommend cloning the repository and installing the package locally:
 
 ```shell
 # Download (git clone)
 git clone https://github.com/M-ERA-NET-MEDIATE/ds-entities-service.git
 cd ds-entities-service
 
-# Install (using pip)
-python -m pip install -U pip
+# Install (local clone)
+export PIP_INDEX_URL=https://gitlab.sintef.no/api/v4/projects/17883/packages/pypi/simple
```

Comment:
If we could avoid this, it would be good I think

Reply:
The only way to avoid this is to publish directly to the public PyPI index instead of SINTEF's GitLab.